helpers/handleBeaconChainContractDeploy.ts

```diff
@@ -0,0 +1,7 @@
+import { Signer } from "@ethersproject/abstract-signer";
+
+export async function handleBeaconChainContractDeploy(deploy: Function, deployer: string) {
+
+}
+
+export default handleBeaconChainContractDeploy
```

Comment:
✂️  this file, looks like it.  No placeholders!

Reply:
Fair point, but also trying to understand how I want to organize everything. Still need to reactor the Rinkeby function into that "placeholder".

Reply:
Placeholder filled 😏

Reply:
This file is still empty....?  Should be deleted.  Dead code